data/compatibility/supportStatus.yml

```
@@ -5,13 +5,14 @@
   supported: "No, development only"
   releaseDate:
   eolDate:
-  k8sVersions: ["1.25", "1.26", "1.27", "1.28"]
+  k8sVersions: ["1.25", "1.26", "1.27", "1.28", "1.29"]
```

I think the change should be something like
```suggestion
  k8sVersions: ["1.26", "1.27", "1.28", "1.29"]
```

---

Yeah looks like 1.25 is EOL. I haven't removed it from testing yet, but can do that in a follow up.

---

Eh actually, we still declare support for it, so we should probably continue to test it

---

Although, we should remove 1.25 from the supported versions for master moving it to tested.